docs/test-container.md

```diff
@@ -0,0 +1,127 @@
+
+* `-i` to provide a name to a custom TNF container image. Supports local images, as well as images from external registries.
+
+
+* `-k` to set a path to one or more kubeconfig files to be used by the container to authenticate with the cluster. Paths must be separated by a colon.
+
+!!! note
+
+    If `-k` is not specified, autodiscovery is performed.
+
+    The autodiscovery first looks for paths in the `$KUBECONFIG` environment variable on the host system, and if the variable is not set or is empty, the default configuration stored in `$HOME/.kube/config` is checked.
+
+* `-n` to give the network mode of the container. Defaults set to `host`, which requires selinux to be disabled. Alternatively, `bridge` mode can be used with selinux if TNF_CONTAINER_CLIENT is set to `docker` or running the test as root.
+
+!!! note
+
+    See the [docker run --network parameter reference](https://docs.docker.com/engine/reference/run/#network-settings) for more information on how to configure network settings.
+
+
+**Command to run**
+
+```shell
+./run-tnf-container.sh -k ~/.kube/config -t ~/tnf/config
+-o ~/tnf/output -l "networking,access-control"
+```
+
+See [General tests](#general-tests) for a list of available keywords.
+
+### Run with `docker`
+
+By default, `run-container.sh` utilizes `podman`. However, an alternate container virtualization
+client using `TNF_CONTAINER_CLIENT` can be configured. This is particularly useful for operating systems that do not readily support
+`podman`.
+
+In order to configure the test harness to use `docker`, issue the following prior to
+`run-tnf-container.sh`:
+
+```shell
+export TNF_CONTAINER_CLIENT=docker
+```
+
+## With local test container image
+
+
+### Build locally
+
+```shell
+podman build -t cnf-certification-test:v1.0.5 \
+  --build-arg TNF_VERSION=v1.0.5 \
+  --build-arg OPENSHIFT_VERSION=4.7.55 .
+```
+
+  - `TNF_VERSION` value is set to a branch, a tag, or a hash of a commit that will be installed into the image
+  -  `OPENSHIFT_VERSION` value points to the OCP version of the cluster in which the workloads to be tested are deployed.
+
+
+### Build from an unofficial source
+
+The unofficial source could be a fork of the TNF repository.
+
+Use the `TNF_SRC_URL` build argument to override the URL to a source repository.
+
+```shell
+podman build -t cnf-certification-test:v1.0.5 \
+  --build-arg TNF_VERSION=v1.0.5 \
+  --build-arg TNF_SRC_URL=https://github.com/test-network-function/cnf-certification-test \
+  --build-arg OPENSHIFT_VERSION=4.7.55 .
+```
+
+### Run the tests
+
+Specify the custom TNF image using the `-i` parameter.
+
+```shell
+./run-tnf-container.sh -i test-network-function:v1.0.5
```

Comment:
We could probably make this a more recent version.  Also I think the name of the image should be changed to match the current repo/image name.

```suggestion
./run-tnf-container.sh -i cnf-certification-test:v4.0.2
```